### PR TITLE
debian: remove some obsolete deps

### PR DESCRIPTION
**Pull request by Dudemanguy**
Neither of these are needed.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -27,15 +27,13 @@ Build-Depends:
  libfribidi-dev,
  libgl1-mesa-dev,
  libgbm-dev,
- libgme-dev,
```
Comment (sfan5) [inline debian/control:30]: https://github.com/mpv-player/mpv-build/blob/f667554e23d754c0619603a1b88fe8d49a56318a/debian/rules#L44

but I agree it's kind of useless. just like `libgsm`

File: debian/control [hunk 2]
```diff
@@ -27,15 +27,13 @@ Build-Depends:
...
  libgnutls-dev | libgnutls28-dev,
  libgsm1-dev,
  libharfbuzz-dev,
  libjack-jackd2-dev | libjack-dev,
  libjpeg-dev,
  libbrotli-dev,
```
Comment (sfan5) [inline debian/control:35]: this is either a transitive dep or unused, too

Comment (Dudemanguy) [reply debian/control:35]: Allegedly needed by freetype: https://github.com/mpv-player/mpv-build/commit/215994da1865d9c6bfb455c67ed8cd7c683f0221

Comment (Dudemanguy) [reply debian/control:34]: I don't know how building apt packages works. ¯\_(ツ)_/¯

Comment (sfan5) [reply debian/control:34]: me neither but I assume you'd test this before merging and if I was wrong it would be noticed

**Pull request merged**